src/Modules/OrchardCore.Commerce/Assets/Styles/shopping-cart-widget.scss

```diff
@@ -0,0 +1,19 @@
+.widget-shopping-cart-widget {
+    position: absolute;
+    right: 10px;
+    top: 10px;
+}
+
+.shopping-cart-link {
+    position: relative;
+
+    // Making sure widget is always visible.
+    z-index: 9999;
+}
+
+.item-count {
```

Comment:
Looks cool, but the number is not quite centered.
![image](https://user-images.githubusercontent.com/4007293/188462976-e7a2fbc3-8284-4a03-a4d5-74ad72a70769.png)
Set the `line-height` to the `height`, see [here](https://stackoverflow.com/a/19613616/14748624).